src/Objects/MonitorManager.vala

```diff
@@ -206,13 +216,19 @@ public class Display.MonitorManager : GLib.Object {
         }
         if (virtual_monitor_number != monitor_number) {
             // Create Inactive VirtualMonitors for each disabled monitor.
-            virtual_monitors.add_all(create_missing_virtual_monitors(false, max_scale));
+            virtual_monitors.add_all (create_missing_virtual_monitors (false, max_scale));
```

Comment:
Shouldn't this change in size be notified?

Reply:
I'm also not sure what you mean here either.
These are newly created VirtualMonitor instances, so there aren't corresponding DisplayWidget to notify yet?

Reply:
OK, but see previous comment about connecting differently.